ui/decredmaterial/theme.go

```diff
@@ -74,9 +74,8 @@ func NewTheme(fontCollection []text.FontFace, decredIcons map[string]image.Image
 }
 
 func (t *Theme) SwitchDarkMode(isDarkModeOn bool, decredIcons map[string]image.Image) {
-	t.Color.DefualtThemeColors()
+	t.Color = t.Color.DefaultThemeColors()
```

Comment:
I dont see the reason for this change. if it does not affect your code in anyway, kindly revert.

Reply:
The DefualtThemeColors function should return object of default colors, it shouldn't is change current colors of theme because some page in app use this function to get default color

Reply:
@JustinBeBoy perhaps you can create a new issue for this, and have it solved in a separate PR, properly documenting the reason for the change. As the added change does not affect this PR/ is not relating to this PR, it's outside the scope of this PR.

Reply:
oh I see, thanks for pointing that out, i thought it was just some code optimization